src/olympia/versions/models.py

```diff
@@ -236,6 +236,23 @@ def delete(self, hard=False):
             self.save()
 
     @property
+    def is_user_disabled(self):
+        return self.files.filter(status=amo.STATUS_DISABLED).exclude(
+            original_status=amo.STATUS_NULL).count()
+
+    @is_user_disabled.setter
+    def is_user_disabled(self, disable):
+        # User wants to disable (and the File isn't already).
+        if disable:
+            for f in self.files.exclude(status=amo.STATUS_DISABLED).all():
```

Comment:
hmm, I thought it was in there but I'm mistaken. It's just a recommendation and pylint errors out on that.